legal-api/tests/unit/reports/templates/test_name_translations.py

```diff
@@ -0,0 +1,56 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Name Translation template tests."""
+
+from typing import Final
+from pathlib import Path
+
+from flask import current_app
+from jinja2 import Template
+
+
+title: Final = 'Company Name Translation(s)'
+
+
+def get_template():
+    """Returns the template."""
+    template_path = current_app.config.get('REPORT_TEMPLATE_PATH')
+    template_code = Path(f'{template_path}/template-parts/common/nameTranslation.html').read_text()
+    return Template(template_code)
+
+
+def test_no_rendering(session):
+    """Test Company Name Translation(s) section should not rendered when no current translation nor previous ones."""
+    template = get_template()
+    rendered = template.render(listOfTranslations=[], previousNameTranslations=[])
+    assert title not in rendered
+
+
+def test_render_none(session):
```

Comment:
Could these be collapsed to a parameterized test?

Reply:
They could but I don't think we should do that for the sake of simplicity and readability. If you really want me to do that I can but I don't like the way become being parameterized.

Reply:
i think it would make it easier for the next person to add if needed, as they look pretty similar to me.
you can leave it and the next person can do that ...